Sources/Core/Formatter/SinglelineFormatter.swift

```diff
@@ -15,6 +15,20 @@ class SinglelineFormatter: PrettyFormatter {
         return "[\(contents)]"
     }
 
+    func tupleString(elements: [(String?, String)]) -> String {
+        // if the labels of tuples are not specificated, they are like ".1" (not nil).
+        // Using "." as the first charactor of the label of tuple is prohibited.
+        let labelValuePairs: [String] = elements.map { label, value in
+            if let label = label, label.first != "." {
```

Comment:
I think a better way would be this judge move to `Pretty.swift`.

For example:

```swift
// Pretty.swift
case .tuple:
    let elements: [(String?, String)] = mirror.children.map {
        var labelOrNil: String? = nil
        if let label = $0.label, label.first != "." {
            labelOrNil = label
        }
        return (label: labelOrNil, value: _string($0.value))
    }
    return formatter.tupleString(elements: elements)

// Here
let labelValuePairs: [String] = elements.map { label, value in
    if let label = label {
...
```

Formatter should be concentrate styling therefore should not judge property of data.

Reply:
Thank you for a comment!

This suggestion is so reasonable, I adopt it in